xonsh/proc.py

```diff
@@ -552,8 +550,6 @@ def __bool__(self):
     def __eq__(self, other):
         return self.returncode == other
 
-    def __hash__(self):
-        return hash(self.returncode)
```

Comment:
I wouldn't mind adding a `rtn` property to `CompletedCommand` to shorten this up.